router/core/router.go

```diff
@@ -1355,6 +1285,39 @@ func (r *Router) Start(ctx context.Context) error {
 	return nil
 }
 
+type UsageTrackerNoOp struct{}
+
+func (u *UsageTrackerNoOp) TrackExecutionConfigUsage(_ map[string]any) {}
+
+func (u *UsageTrackerNoOp) TrackRouterConfigUsage(_ map[string]any) {}
+
+func (u *UsageTrackerNoOp) Close() {}
+
+func (u *UsageTrackerNoOp) TrackUptime(_ context.Context) {}
+
+func (r *Router) configureUsageTracking(ctx context.Context) (err error) {
+	if !r.enableUsageTracking {
+		r.usage = &UsageTrackerNoOp{}
+		r.logger.Info("Cosmo Usage Tracking is disabled. (dev build)")
```

Comment:
Why `dev build`?

Reply:
we only enable tracking when the version of the build is not "dev"